.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-  - "node"
```

Comment:
Why remove this, this will quickly tell us if the new version of node can compile it.

Reply:
Oh, word, I removed 6 originally and was trying to put it back together